package/MDAnalysis/auxiliary/base.py

```diff
@@ -1,55 +1,112 @@
 import numpy as np
+import math
 
 class AuxReader(object):
-    """ Base class for auxiliary readers
-    Does iteration and aligning between trajectory ts/auxiliary step
-    Actual reading/parsing of data to be handled by individual readers.
-    """
+    """ Base class for auxiliary readers.
+
+    Handles iteration and aligning between trajectory timesteps and auxiliary 
+    step.
+
+    Reading/parsing of data should be handled by individual format-specific 
+    readers.
+
+    When reading in a trajectory, auxiliary data steps are 'assigned' the 
+    closest trajectory timestep.
+
+    Attributes:
+      step: 
+        number of the current auxiliary step, starting at 0
+      dt:   
+        change in time between auxiliary steps (ps)
+      initial_time: 
+        time of first auxilairy step (ps)
+      _data:
```

Comment:
Private attributes (those prefixed with `_`) should not appear in the docstring as they are not part of the exposed API. They should be documented with regular comments.